src/main/kotlin/com/fasterxml/jackson/module/kotlin/KotlinFeature.kt

```diff
@@ -0,0 +1,19 @@
+package com.fasterxml.jackson.module.kotlin
+
+import com.fasterxml.jackson.core.util.JacksonFeature
+import java.util.*
+
+enum class KotlinFeature(private val enabledByDefault: Boolean, private val mask: Int) : JacksonFeature {
+
+    NullToEmptyCollection(enabledByDefault = false, mask = 1),
```

Comment:
Not sure if Kotlin enums work the same as Java ones, but java has `Enum.ordinal()` which can be used to get mask  (`1 << enum.ordinal()`). Avoids one class of potential problems.